beta/src/pages/learn/passing-data-deeply-with-context.md

```diff
@@ -205,7 +205,7 @@ But how can the `<Heading>` component know the level of its closest `<Section>`?
 You can't do it with props alone. This is where context comes into play. You will do it in three steps:
 
 1. **Create** a context. (You can call it `LevelContext`, since it's for the heading level.)
-2. **Use** that context from the component that needs the data. (`Header` will use `LevelContext`.)
+2. **Use** that context from the component that needs the data. (`Heading` will use `LevelContext`.)
```

Comment:
I see that we have more occurrences of `Header` across the page? Can we change at other places as well?